Обновить значение по умолчанию "show-pollution-visuals"

Версия 2.0.7, серверное исполнение.
Ошибка в логе: 
   0.255 Error Util.cpp:81: Error while loading mod-setting prototype "show-pollution-visuals" (bool-setting): Value must be a bool in property tree at ROOT.bool-setting.show-pollution-visuals.default_value
Modifications: pollution visuals › Picker Tweaks

Мод Picker Tweaks

Выполняемая команда: 
	data:extend({{
		type = "bool-setting",
		name = "show-pollution-visuals",
		default_value = "true",
		setting_type = "runtime-per-user",
	}})

--- a/mods/SingleColorTerrain/settings.lua
+++ b/mods/SingleColorTerrain/settings.lua
@@ -6,8 +6,7 @@ data:extend(
 type = "bool-setting",
 name = "white-concrete",
 setting_type = "startup",
-default_value = "true",
-allowed_values = {"true", "false"},
+default_value = true,
 localised_name = "White Concrete?",
 order = "e"
 },
@@ -17,8 +16,7 @@ order = "e"
 type = "bool-setting",
 name = "white-stone",
 setting_type = "startup",
-default_value = "true",
-allowed_values = {"true", "false"},
+default_value = true,
 localised_name = "White Stone Path?",
 order = "d"
 },
@@ -28,8 +26,7 @@ order = "d"
 type = "bool-setting",
 name = "no-decor",
 setting_type = "startup",
-default_value = "false",
-allowed_values = {"true", "false"},
+default_value = false,
 localised_name = "No Decorations?",
 order = "b"
 },
@@ -41,8 +38,7 @@ order = "b"
 type = "bool-setting",
 name = "no-clouds",
 setting_type = "startup",
-default_value = "true",
-allowed_values = {"true", "false"},
+default_value = true,
 localised_name = "No Clouds?",
 order = "c"
 },

--- a/mods/pollution-visuals/settings.lua
+++ b/mods/pollution-visuals/settings.lua
@@ -2,7 +2,7 @@ data:extend({
 	{
 		type = "bool-setting",
 		name = "show-pollution-visuals",
-		default_value = "true",
+		default_value = true,
 		setting_type = "runtime-per-user",
 	},
 

